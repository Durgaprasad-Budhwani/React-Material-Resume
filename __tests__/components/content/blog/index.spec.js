/**
 * Created by Durgaprasad Budhwani on 10/9/2017.
 */
/* global expect,it, describe, afterEach */
import React from 'react';
import BlogContainer from '../../../../src/components/content/blog';
import renderer from 'react-test-renderer';
import {mount} from 'enzyme'
import fetchMock from 'fetch-mock'

describe("Blog Render", () => {
    afterEach(() => {
        fetchMock.restore();
    });
    
    it('renders blog container', () => {
        let blogs = [{image: 'image', title: "title", day: "day", month: "month"}];
        fetchMock.mock('blogs.json', blogs);
        const tree = renderer.create(
            <BlogContainer
                top={10}
                height={120}
            />
        );
        let instance = tree.getInstance();
        expect(tree.toJSON()).toMatchSnapshot();
        expect.arrayContaining(instance.state);
    });
});
