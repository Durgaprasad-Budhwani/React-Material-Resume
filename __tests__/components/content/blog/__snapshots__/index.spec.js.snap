// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog Render renders blog container 1`] = `
<div
  className=""
>
  <span
    className="SectionTitle-sectionTitle-92"
  >
    Articles
  </span>
  <div
    align="stretch"
    className="MuiGrid-typeContainer-3 MuiGrid-typeItem-4 MuiGrid-spacing-xs-8-23 MuiGrid-justify-xs-space-between-21 MuiGrid-grid-xs-12-39"
  />
</div>
`;
